### ISSUE TITLE
System - Undoc 6.0 APIs

### ISSUE THREAD

**Issue by carlossanlop** [dotnet/runtime#54882]
The following APIs were introduced in 6.0 and did not get documented in triple slash. They will need to be manually documented directly in dotnet-api-docs, and must target the net6-rc1 branch. This needs to be done before Tuesday September 14th so we have enough time to generate the IntelliSense package that will be shipped to GA.

Update: I found _some_ APIs that could be automatically ported, and I submitted them here: https://github.com/dotnet/dotnet-api-docs/pull/7106 and I removed them from the list below.

| Undoc API                                                                                                                   |
|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| [M:System.Math.Sign(System.IntPtr)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/Math.xml)                                                                                                                                                                          |
| [M:System.Math.SinCos(System.Double)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/Math.xml)                                                                                                                                                                        |
| [M:System.MathF.SinCos(System.Single)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MathF.xml)                                                                                                                                                                      |

<details>
  <summary>Already addressed</summary>

| Undoc API                                                                                                                   |
|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| [M:System.MemoryExtensions.EnumerateLines(System.ReadOnlySpan{System.Char})](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions.xml)                                                                                                                     |
| [M:System.MemoryExtensions.EnumerateLines(System.Span{System.Char})](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions.xml)                                                                                                                             |
| [M:System.MemoryExtensions.SequenceEqual\`\`1(System.Span{\`\`0},System.ReadOnlySpan{\`\`0},System.Collections.Generic.IEqualityComparer{\`\`0})](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions.xml)                                                        |
| [M:System.MemoryExtensions.SequenceEqual\`\`1(System.ReadOnlySpan{\`\`0},System.ReadOnlySpan{\`\`0},System.Collections.Generic.IEqualityComparer{\`\`0})](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions.xml)                                                |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.#ctor(System.Int32,System.Int32,System.Span{System.Char},System.IFormatProvider,System.Boolean@)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml) |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.#ctor(System.Int32,System.Int32,System.Span{System.Char},System.Boolean@)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                        |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendFormatted(System.String,System.Int32,System.String)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                        |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendFormatted(System.ReadOnlySpan{System.Char},System.Int32,System.String)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                     |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendFormatted(System.ReadOnlySpan{System.Char})](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                                |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendFormatted(System.Object,System.Int32,System.String)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                        |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendFormatted(System.String)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                                                   |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendFormatted\`\`1(\`\`0,System.Int32,System.String)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                               |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendFormatted\`\`1(\`\`0,System.String)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                                            |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendFormatted\`\`1(\`\`0)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                                                          |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendFormatted\`\`1(\`\`0,System.Int32)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                                             |
| [M:System.MemoryExtensions.TryWriteInterpolatedStringHandler.AppendLiteral(System.String)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                                                     |
| [T:System.MemoryExtensions.TryWriteInterpolatedStringHandler](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/MemoryExtensions+TryWriteInterpolatedStringHandler.xml)                                                                                                  |
| [M:System.String.ReplaceLineEndings(System.String)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/String.xml)                                                                                                                                                        |
| [M:System.UIntPtr.Parse(System.ReadOnlySpan{System.Char},System.Globalization.NumberStyles,System.IFormatProvider)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/UIntPtr.xml)                                                                                       |
| [M:System.UIntPtr.TryParse(System.ReadOnlySpan{System.Char},System.UIntPtr@)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/UIntPtr.xml)                                                                                                                             |
| [M:System.UIntPtr.TryParse(System.ReadOnlySpan{System.Char},System.Globalization.NumberStyles,System.IFormatProvider,System.UIntPtr@)](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/UIntPtr.xml)                                                                    |
| [F:System.Uri.UriSchemeFtps](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/Uri.xml)                                                                                                                                                                                  |
| [F:System.Uri.UriSchemeSftp](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/Uri.xml)                                                                                                                                                                                  |
| [F:System.Uri.UriSchemeSsh](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/Uri.xml)                                                                                                                                                                                   |
| [F:System.Uri.UriSchemeTelnet](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/Uri.xml)                                                                                                                                                                                |
| [F:System.Uri.UriSchemeWs](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/Uri.xml)                                                                                                                                                                                    |
| [F:System.Uri.UriSchemeWss](https://github.com/dotnet/dotnet-api-docs/blob/net6-rc1/xml/System/Uri.xml)                                                                                                                                                                                   |

</details>

**Comment by carlossanlop** [comment]
@dotnet/area-system-numerics It seems some of the APIs here might belong to Numerics. If this is a correct assumption, can you please take care of them? If not, who do you think should?

**Comment by carlossanlop** [comment]
Submitted a PR for some of the IntPtr and MemoryExtensions APIs that did have documentation to port. https://github.com/dotnet/dotnet-api-docs/pull/7131

**Comment by carlossanlop** [comment]
I took care of most of the System APIs in the list.

@tannergooding as discussed via chat, you'll take a look at the 3 new System.Math APIs.

Note: Early morning tomorrow we will get the net6-rc1 branch merged into main in dotnet-api-docs, so your PR will have to target main directly.
